person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  # rubocop:disable Naming/PredicateName
+  def is_of_age?
+    @age >= 18
+  end
+  # rubocop:enable Naming/PredicateName
+
+  public
+
+  def can_use_services?
+    this.is_of_age? or @parent_permission
+  end
```

Comment:
- [ ] In Ruby programming language, you only need to call the class method without 'this'.
- [ ] Following Ruby best practices, it would be nicer to use `| |` instead of `or`.
```ruby
  def can_use_services?
     is_of_age? | | @parent_permission
  end
```